numbers/distribution.go

```diff
@@ -261,14 +261,22 @@ func BinomialLeftSummation(n int, k int, p float64, logOutput bool) float64 {
 
 // BinomialRightSummation calculates the sum of binomial probabilities to the right of k successes for a binomial distribution with n experiments and a success probability of p, inclusive.
 func BinomialRightSummation(n int, k int, p float64, logOutput bool) float64 {
+	var val, sig, x float64
 	if k == 0 {
 		if logOutput {
 			return 0
 		} else {
 			return 1
 		}
 	}
-	return evaluateRightBinomialSum(n, k, p, logOutput)
+	if float64(n)*p > 5 && float64(n)*(1-p) > 5 {
+		val = float64(n) * p * (1 - p)
+		x = float64(k) - 0.5
+		sig = math.Sqrt(float64(n) * val)
+		return NormalDist(x, val, sig)
```

Comment:
Note to self: when the number of tests is high enough and the probability of success is low enough, can model binomial distribution with normal distribution. Using the existing method to calculate the area under the normal distribution curve, can quickly calculate the proxy for "the area under the binomial distribution curve"